src/main/java/com/synopsys/integration/detect/lifecycle/run/step/IntelligentModeStepRunner.java

```diff
@@ -82,7 +82,8 @@ public void runOnline(
         BdioResult bdioResult,
         NameVersion projectNameVersion,
         DetectToolFilter detectToolFilter,
-        DockerTargetData dockerTargetData
+        DockerTargetData dockerTargetData,
+        Set<String> scanTargets
```

Comment:
Instead of scanTargets, can we name the variable binaryTargets to keep it consistent?

Reply:
Done, thanks